metabase/metabase
# Filter timelines by question's dates

Epic: https://github.com/metabase/metabase/issues/20289

Changes:
- Filters events by question's dates
- Adds toasts when creating and updating events from the query builder

How to test:
- Go to a collection and create a timeline with events for years 2016-2021
- Go to a timeseries question and open the event sidebar
- Make sure that only events that are within the question's range are displayed in the sidebar
